### DISCUSSION TITLE
Can an FPGA Board Fully Control a 3D Printer?

### DISCUSSION THREAD

**Discussion by Eric12368** [micropython/micropython#15487] [General]
I'm currently working on an experiment to see if an FPGA board can be used to fully control a 3D printer. I found this interesting article that discusses the potential of using FPGA boards for 3D printer control: [Could an FPGA Board Be Used to Fully Control a 3D Printer?](https://www.icdrex.com/could-an-fpga-board-be-used-to-fully-control-a-3d-printer/).

Has anyone here tried something similar or have any insights on this? Any advice or experiences would be greatly appreciated!

**Comment by peterhinch** [comment]
This doesn't seem to have any relevance to MicroPython.
